### PR TITLE
6.8.7 release notes

### PR DESCRIPTION
**Pull request by jakelandis**

### CODE REVIEW

File: docs/reference/release-notes/6.8.asciidoc [hunk 1]
```diff
@@ -11,6 +9,36 @@ Machine Learning::
...
+
+Search::
+* Fix NPE bug inner_hits {pull}50709[#50709] (issue: {issue}50539[#50539])
+
+[[enhancement-6.8.7]]
+[float]
+=== Enhancements
+
+SQL::
+* SQL: Verify Full-Text Search functions not allowed in SELECT {pull}51568[#51568] (issue: {issue}47446[#47446])
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc]:
```suggestion
* Verify Full-Text Search functions not allowed in SELECT {pull}51568[#51568] (issue: {issue}47446[#47446])
```

File: docs/reference/release-notes/6.8.asciidoc [hunk 2]
```diff
@@ -26,7 +54,15 @@ Features/Watcher::
...
 
+[[upgrade-6.8.7]]
+[float]
+=== Upgrades
+
+Infra/Packaging::
+* Upgrade the bundled JDK to JDK 13.0.2 {pull}51511[#51511]
 
+Network::
+* Upgrade to Netty 4.1.43 {pull}49798[#49798]
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc]: It looks like the 'Upgrades' section was accidentally placed under the 6.8.6.

Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc]: It looks like this PR was closed but not merged.

```suggestion
```

File: docs/reference/release-notes/6.8.asciidoc (hunk 1 above)
Comment (jrodewig) [reply docs/reference/release-notes/6.8.asciidoc]: Fixed with 3782e9ff9a33354a57e9a2730eaf31bd2e1ffb4d.

File: docs/reference/release-notes/6.8.asciidoc (hunk 2 above)
Comment (jrodewig) [reply docs/reference/release-notes/6.8.asciidoc]: Fixed with 3782e9ff9a33354a57e9a2730eaf31bd2e1ffb4d.

Comment (jrodewig) [reply docs/reference/release-notes/6.8.asciidoc]: Fixed with 3782e9ff9a33354a57e9a2730eaf31bd2e1ffb4d.

**Review by jrodewig — APPROVED** [review]
Pushed 3782e9ff9a33354a57e9a2730eaf31bd2e1ffb4d and a34515a6b2b792b6022da84536bbbe0abf42433e to fix some structural issues.

With those, this LGTM. Thanks!

**Comment by jakelandis** [comment]
Thanks @jrodewig !

**Pull request merged**